Tools-Override/tests.targets

```diff
@@ -214,6 +214,45 @@
       <PostExecutionTestCommandLines Include="copy /y testResults.xml %EXECUTION_DIR%\" />
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+      <_Results_Path>$([System.Environment]::GetFolderPath(SpecialFolder.MyDocuments))\TestResults\</_Results_Path>
+    </PropertyGroup>
+  
+    <!-- The results path is different on the helix machine than locally -->
+    <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true' and '$(ArchiveTests)' == 'true'">
+      <_Results_Path>%USERPROFILE%\Documents\TestResults\</_Results_Path>
```

Comment:
The reason why they are different is because the documents folder might be on a different drive. This is possible on the local machine, but the cloud machines don't do that.

The file name will have the name of the test in them... so they should not collide.